doc/source/quickstart-ios.rst

```diff
@@ -0,0 +1,151 @@
+.. _quickstart-ios:
+
+
+Quickstart iOS
+==================
+
+In this tutorial we will learn how to train a Neural Network on MNIST using Flower and CoreML. 
+
+First of all, it is recommended to create a virtual environment and run everything within a `virtualenv <https://flower.dev/docs/recommended-env-setup.html>`_. 
+
+Our example consists of one *server* and *clients* that all have the same model. 
+
+*Clients* are responsible for generating individual weight-updates for the model based on their local datasets. 
+These updates are then sent to the *server* which will aggregate them to produce a better model. Finally, the *server* sends this improved version of the model back to each *client*.
+A complete cycle of weight updates is called a *round*.
+
+Now that we have a rough idea of what is going on, let's get started. We first need to install Flower. You can do this by running :
+
+.. code-block:: shell
+
+  $ pip install flwr
+
+Or simply install all dependencies using Poetry:
+
+.. code-block:: shell
+
+  $ poetry install
```

Comment:
```suggestion
  $ poetry add flwr
```

Reply:
Got the point, changed it back to poetry add flwr and pip install flwr.